community/blog/2020.10-Introducing-message-handling-with-Azure-Service-Bus-message-influence/README.md

```diff
@@ -0,0 +1,106 @@
+
+```csharp
+public class OrderMessageHandler<Order> : AzureServiceBusMessageHandler<Order>
+{
+    public override async Task ProcessMessageAsync(Order order, AzureServiceBusMessageContext context, ...)
+    {
+        if (order.Customer is null)
+        {
+            // Calling base operation where the Azure Service Bus operations are located.
+            await base.AbandonMessageAsync();
+        }
+        else
+        {
+            // Processing order.
+        }
+    }
+}
+```
+
+Note that in the regular message handling you don't have to specify which message to abandon. This is all done behind the screens so the message handler only has to focus on the message processing and call the specific Azure Service Bus message operations when necessary.
+
+### Fallback message handling
+
+The fallback message handling is possible more correct when we're talking about dead lettering a message. Fallback message handlers will process the Azure Service Bus message when none other regular message handler was able to correctly process it.
+At the end of the fallback message processing, the message could be dead lettered on Azure for example.
+
+Here's an example of how a normal `IServiceBusFallbackMessageHandler` is implemented:
+
+```csharp
+public class OrderFallbackMessageHandler : IAzureServiceBusFallbackMessageHandler
+{
+    public async Task ProcessMessageAsync(Message orderMessage, AzureServiceBusMessageContext context, ...)
+    {
+        // Processing order.
+    }
+}
+```
+
+When you decide you want to dead letter the message, you'll have to implement  `AzureServiceBusFallbackMessageHandler` instead.
+
+```csharp
+public class OrderFallbackMessageHandler : AzureServiceFallbackMessageHandler
+{
+    public override async Task ProcessMessageAsync(Message orderMessage, AzureServiceBusMessageContext context, ...)
+    {
+        // Processing order.
+        await base.DeadLetterAsync(orderMessage);
+    }
+}
+```
+
+Note that because the fallback message handlers have more control over the original Azure Service Bus message, they will have to pass along the message to the base operations.
+
+## Is that all?
+
+No, of course not. The v0.4 and v0.5 version also has some minor updates regarding less verbose logging and some extra correlation extensions to pass along your own custom operation/transation ID property. All are helping in making the developer and consumer more in control of our messaging functionality.
+
+## What's on the horizon?
+
+We'll see how we can reuse our message handling system in the context of Azure functions where the function binding will take the place of the message pump.
+We'll also look at how we can let the consumer be more in control of how the incoming Azure Service Bus messages are being deserialized before they're passed to the message handling.
+
+Have a question, remark, feedback or idea for a superdub addition?
+Feel free to take a look at our GitHub repository and create an issue.
```

Comment:
Here, maybe link to the repo ?

Reply:
Yes, added that. Thanks. Also added the link to create an issue.